packages/hooks/src/useHasSwapFreeAccount.ts

```diff
@@ -0,0 +1,15 @@
+import { useStore } from '@deriv/stores';
+
+// This hook is used to check if the client has a Swap-Free account
+// It checks for availability of market_type 'all' in trading_platform_available_accounts API response
```

Comment:
```suggestion
**
 * This hook is used to check if the client has a Swap-Free account
 * It checks for availability of market_type 'all' in trading_platform_available_accounts API response
 */
```
Please use `JSDoc` 🙏🏻  Otherwise, the documentation won't work.

<img width="526" alt="image" src="https://user-images.githubusercontent.com/72082844/234453823-a046378d-520c-462c-8d95-af28144fcbc6.png">

Reply:
Sure